build.gradle

```diff
@@ -51,18 +51,13 @@ subprojects {
     apply plugin: "jacoco-report-aggregation"
     apply plugin: "groovy"
 
-    tasks.withType(JavaCompile) {
+    tasks.withType(JavaCompile).configureEach {
         options.compilerArgs << "-Xlint:unchecked"
         options.compilerArgs << "-Xlint:deprecation"
     }
 
-    project(":polaris-service") {
-        apply plugin: "application"
-    }
-
-    project(":polaris-core") {
-        apply plugin: "java-library"
-    }
+    // Note: polaris-core seemed to be for Java 11, but uses functions that are introduced in later Java versions
```

Comment:
then shouldn't we just change the version? I'm confused why we would have this workaround?